src/render/attrs/index.js

```diff
@@ -10,6 +10,42 @@ const attrsNotToRender = {
   dangerouslySetInnerHTML: true
 };
 
+const attrsWithExplicitBoolValue = {
+  "aria-expanded": true,
+  "aria-haspopup": true
+};
```

Comment:
@mijamo This is perfect, I will integrate today or tomorrow. That should be the last change on this PR I think @divmain